onnx/onnxifi_ext.h

```diff
@@ -219,6 +219,109 @@ ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI onnxSetIOAndRunGraph(
     const onnxTensorDescriptorV1* outputDescriptors,
     onnxMemoryFenceV1* outputFence);
 
+/*
+ * Tag for version 1 of Caffe2 quantized tensor descriptor structure
+ * (onnxCaffe2QTensorDescriptorV1).
+ *
+ * The tag is unique for this tensor descriptor structure. If ONNXIFI introduce
+ * a new version of the tensor descriptor structure in the future, it will get
+ * a new tag value.
+ *
+ * This tag is a part of fb_caffe2_model extension.
+ * Backends which support this tag MUST list "fb_caffe2_model" in the
+ * result of onnxGetBackendInfo with ONNXIFI_BACKEND_EXTENSIONS information
```

Comment:
does `ONNXIFI_BACKEND_EXTENSIONS` represent just a single key-value?

Reply:
also curious about this, would be good to be able to distinguish between supported extensions right?

Reply:
`ONNXIFI_BACKEND_EXTENSIONS` is a space-separated list of values